virt/virtual_machines/importing_vms/virt-importing-virtual-machine-images-datavolumes.adoc

```diff
@@ -25,6 +21,10 @@ Refer to the operating system documentation for details.
 xref:../../../virt/virtual_machines/importing_vms/virt-tls-certificates-for-dv-imports.adoc#virt-adding-tls-certificates-for-authenticating-dv-imports_virt-tls-certificates-for-dv-imports[included in a ConfigMap]
 in the same namespace as the DataVolume and referenced in the DataVolume configuration.
 
+* To import a container disk: 
+** You may need to xref:../../../virt/virtual_machines/virtual_disks/virt-using-container-disks-with-vms.adoc#virt-preparing-container-disk-for-vms_virt-using-container-disks-with-vms[prepare a container disk from a virtual machine image] and store it in your container registry before importing it.
+** If the container registry does not have TLS security you must xref:../../../virt/virtual_machines/virtual_disks/virt-using-container-disks-with-vms.adoc#virt-disabling-tls-for-registry_virt-using-container-disks-with-vms[add the registry to the `cdi-insecure-registries` ConfigMap] before you can import a container disk from it.
```

Comment:
Add a comma after "TLS security"

(and please see my previous comment re: "TLS security" itself - same for wherever else this phrase appears)

(and ConfigMap)